src/vm/database/marf.rs

```diff
@@ -294,6 +294,27 @@ impl MarfedKV {
         }
     }
 
+    pub fn begin_read_only_checked<'a>(
```

Comment:
Can you DRY `begin_read_only<'a>` by calling `.expect()` on this new function?

Reply:
Fixed in 1d977c1